src/app/search/page.tsx

```diff
@@ -16,42 +18,24 @@ const ProductsList = async () => {
         personalize={product.personalize.join(", ")}
         price={product.price}
         restaurant={product.restaurant ?? ""}
-        // imageUrl={product.imageUrl}
       />
     </div>
   ));
 };
 
 export default async function Search() {
-  const products = await api.product.getAll.query();
-  //mutation para crear productos
-  // const productCreationResponse = await api.product.create.mutate({
-  //   name: "Sample Product Name",
-  //   description: "This is a sample description of the product.",
-  //   calification: "Excellent",
-  //   personalize: ["Customization Option 1", "Customization Option 2"],
-  //   price: 199,
-  //   restaurant: "Sample Restaurant Name",
-  // });
-
   return (
-    <main className="w-[calc(100vw - 10rem)] m-20 flex flex-col max-sm:m-10">
-      <div className=" flex  w-60 flex-col gap-3.5 max-sm:hidden">
-        <h6 className="text-[20px] font-normal leading-[108%] tracking-tight text-gray-900">
-          Regalos
-        </h6>
-        <h4 className="text-[35px] font-medium leading-[108%] text-gray-900">
-          Food Box
-        </h4>
-        <p className="text-[18px] font-normal leading-[108%] tracking-tight text-gray-400">
-          Más de 30 opciones
-        </p>
+    <main className="w-[calc(100vw - 10rem)] m-10 flex flex-col max-sm:m-10">
+      <div className="flex w-60 flex-col gap-3.5 leading-[108%] max-sm:hidden">
```

Comment:
nitpick: `leading-[108%]` seems like something straight out of the Figma. We usually try to adapt the designs to our current design system so we don't have to do specific sizes like this